code/graphics/render.cpp

```diff
@@ -1193,33 +1193,71 @@ static void draw_bitmap_list(bitmap_rect_list* list, int n_bm, int resize_mode,
 	auto vert_list = new vertex[6 * n_bm];
 	float sw = 0.1f;
 
+
+	float rad = 0;
+
+	if (angle != 0) {
+		rad = fl_radians(angle);
+	}
```

Comment:
Lua functions are in radians, so this should be in radians also.

Reply:
Sorry about that. I could swear that degrees were the default. I'll fix it.